4.0.0/alpine/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM alpine:3.6
+MAINTAINER "Philip O'Toole <philip.otoole@yahoo.com>"
+
+# Here we install GNU libc (aka glibc) and set C.UTF-8 locale as default.
+# required to use 'rqlited' within alpine as it is not statically linked
+RUN ALPINE_GLIBC_BASE_URL="https://github.com/sgerrand/alpine-pkg-glibc/releases/download" && \
+    ALPINE_GLIBC_PACKAGE_VERSION="2.23-r3" && \
+    ALPINE_GLIBC_BASE_PACKAGE_FILENAME="glibc-$ALPINE_GLIBC_PACKAGE_VERSION.apk" && \
+    ALPINE_GLIBC_BIN_PACKAGE_FILENAME="glibc-bin-$ALPINE_GLIBC_PACKAGE_VERSION.apk" && \
+    ALPINE_GLIBC_I18N_PACKAGE_FILENAME="glibc-i18n-$ALPINE_GLIBC_PACKAGE_VERSION.apk" && \
+    apk add --no-cache --virtual=.build-dependencies wget ca-certificates && \
+    wget \
+        "https://raw.githubusercontent.com/andyshinn/alpine-pkg-glibc/master/sgerrand.rsa.pub" \
```

Comment:
The glibc is built using this project:
https://github.com/sgerrand/docker-glibc-builder

The source for glibc as indicated by the file `builder` within the project is:
http://ftp.gnu.org/gnu/glibc/glibc-$version.tar.gz

The pub key passed in allows Alpine `apk` to verify the package that is being installed. The private key is used in the actual build process. That is related to the internals of the how the `apk` package manager works.